app/controllers/users_controller.rb

```diff
@@ -5,16 +5,16 @@ def new
 
   def show
     @user = User.find_by id: params[:id]
-    if @user.nil?
-      flash[:danger] = t :id_not_exist
-      redirect_to root_path
-    end
+    return unless @user.nil?
```

Comment:
Không nên dùng 2 lần phủ định bạn nhé 
```suggestion
    return if @user.present?
```

Reply:
ok tks b